promotion/controllers.go

```diff
@@ -233,3 +236,34 @@ func GetClaim(service *Service) handlers.AppHandler {
 
 	})
 }
+
+// AggregateClaims returns the number of claimed grants
+func AggregateClaims(service *Service) handlers.AppHandler {
+	return handlers.AppHandler(func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+		claimType := chi.URLParam(r, "claimType")
+		walletIDParam := r.URL.Query().Get("walletID")
+		walletID, err := uuid.FromString(walletIDParam)
```

Comment:
Please see https://github.com/brave-intl/bat-go/blob/e7aba9f7c1c965bc3533e8f70e7162032826ec8a/promotion/controllers.go#L69 for the preferred way to handle error checking for this query parameter.